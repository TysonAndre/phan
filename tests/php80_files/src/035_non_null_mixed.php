<?php
function decode_or_default(string $json) {
    return json_decode($json) ?? 'default';
}
var_export(decode_or_default('null'));
