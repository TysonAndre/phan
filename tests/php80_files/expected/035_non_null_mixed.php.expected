%s:2 PhanPluginUnknownFunctionReturnType Function \decode_or_default() has no declared or inferred return type (Types inferred after analysis: \stdClass|bool|float|int|list<mixed>|string)
